html-forms/index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Forms</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+    <form action="">
+      <div>
+        Email
+      </div>
+      <div>
+        <label for="email">
+          <input type="text" name="email">
+        </label>
+      </div>
+      <div>
+        <label for="coding-cool">
+          Why is coding cool?
+        </label>
+      </div>
+      <div>
+        <select name="coding">
+          <option value="select">Select an option</option>
+          <option value="interesting">Because it's interesting</option>
+          <option value="fun">Because it's fun</option>
+          <option value="powerful">Because it's powerful</option>
+        </select>
+      </div>
+      <div>
+        <p>Favorite Language (choose one)</p>
+      </div>
+      <div>
+        <input type="radio" name="language" value="html">
```

Comment:
Lines 34, 36, 38 should all have `id`s that correspond to the `for` attribute value in their `label`.